src/Compilers/CSharp/Test/Emit/CodeGen/SwitchTests.cs

```diff
@@ -2623,10 +2623,10 @@ static void Main()
 }";
             var compilation = base.CreateCSharpCompilation(text);
             compilation.VerifyDiagnostics(
-                // (8,18): error CS0150: A constant value is expected
+                // (8,18): error CS9135: A constant value of type 'int?' is expected
```

Comment:
>// (8,18): error CS9135: A constant value of type 'int?' is expected [](http://example.com/codeflow?start=16&length=68)

This is actually an interesting scenario. I think we should strip the nullability from value types used in the message. We have ```StrippedType``` helper for this.

Reply:
Done!